tests/css_indent_styles/indent-styles.scss

```diff
@@ -81,4 +81,9 @@ a::after {
 
 // another comment
 /* whatever 
-do does*/
\ No newline at end of file
+do does*/
+
+div {
+  div.foo {}
+}
+div.foo.bar {} // should only be indented against div not div.foo
```

Comment:
Can you add a newline at then end of this file, please? That'll make the snapshot easier to read.